### PR TITLE
Convert to markdown and translated based on latest es-US content

### PR DESCRIPTION
**Pull request by Malaaaa**

### CODE REVIEW

**Comment by yin1999** [comment]
Looks fine. That'll be better if you could update the title of this pr (more specific, you can use the title in #4714 : `Convert to markdown and translated based on latest es-US content`).

File: files/zh-cn/learn/html/index.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+> 我们已经所有你需要的放在一个课程里，
+> 以努力实现你的目标。
+>
+> [**开始**](/zh-CN/docs/Learn/Front-end_web_developer)
+
+## 须知
+
+在开始这个主题的学习之前，你至少要基本熟悉使用电脑和被动地使用网络（即单纯地查看内容）。你应该设置好一个基础的工作环境，详细参照 [安装基础软件](/zh-CN/docs/Learn/Getting_started_with_the_web/Installing_basic_software), 并且理解如何新建和管理文件，详细参照[文件处理](/zh-CN/docs/Learn/Getting_started_with_the_web/Dealing_with_files)——这两者都在 [Web 入门](/zh-CN/docs/Learn/Getting_started_with_the_web) 的完全新手模块里。
+
+在尝试学习这个主题之前，建议先完成 [Web 入门](/zh-CN/docs/Learn/Getting_started_with_the_web) 不过这并不是绝对必要的。[HTML 基础](/zh-CN/docs/Learn/Getting_started_with_the_web/HTML_basics)里大多数的文章在[HTML 介绍](/zh-CN/docs/Learn/HTML/Introduction_to_HTML)模块中也有涉及，不过要详细得多。。
```
Comment (yin1999) [inline files/zh-cn/learn/html/index.md:32]: Remove the extra `。` in this end.

Comment (yin1999) [reply files/zh-cn/learn/html/index.md:32]: update the document, using `git reset HEAD~1` and then commit with this changes. Or you can commit first and then using `git rebase` to merge the last two commit.

**Review by yin1999 — CHANGES_REQUESTED** [review]

File: files/zh-cn/learn/html/index.md (hunk 1 above)
Comment (irvin) [reply files/zh-cn/learn/html/index.md:32]: merge it for now. you can add another pr to improve above point.

**Pull request merged**